cocos/renderer/CCTextureCache.cpp

```diff
@@ -47,25 +47,32 @@ using namespace std;
 
 NS_CC_BEGIN
 
+static std::string s_etc1AlphaFileSuffix = "Alpha";
```

Comment:
i am not sure if using `Alpha` as suffix for alpha picture's name is good or not, maybe `@alpha` or `_alpha` is better. @ricardoquesada any suggestion?

And it is better to define it as static member variable as `TextureCache`.